pkg/common/common.go

```diff
@@ -1,20 +1,75 @@
 package common
 
 import (
+	"encoding/json"
 	"fmt"
+	"log"
 	"os"
 	"os/exec"
 	"path/filepath"
 
+	k8sReport "github.com/aquasecurity/trivy/pkg/k8s/report"
+	"github.com/aquasecurity/trivy/pkg/types"
+
 	"golang.org/x/exp/slices"
 )
 
 func IsHelp() bool {
 	return slices.Contains(os.Args, "--help") || slices.Contains(os.Args, "-h")
 }
 
-func IsK8s() bool {
-	return slices.Contains(os.Args, "kubernetes") || slices.Contains(os.Args, "k8s")
+func ReadReport(fileName string) (*types.Report, error) {
+
+	log.Println("Read report", fileName)
+	if report, err := ReadJson[k8sReport.Report](fileName); err == nil {
```

Comment:
no, `readjson` isn't a good place for k8s detect. 
for example `os.Open(fileName)` can get permission errors. twice.
also `ReadJson` isn't relevant name.